src/__tests__/not-found.test.js

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+import * as renderer from 'react-test-renderer';
+import NotFound404 from 'pages/NotFound404';
+
+const translationProps = {
+  translation: jest.fn()
+};
```

Comment:
Será que aqui conseguimos ter algum texto com as keys que pretendemos, exemplo `NotFound404-Title` e assim na snapshot veriamos o texto retornado pelo translationProps?

Reply:
Pelo que pesquisei aqui superficialmente, dá sim. Mas eu não consigo fazer isso. To em sprint no trabalho 😢 

https://react.i18next.com/misc/testing#example-test-using-this-configuration

Reply:
Sem stress @juliamendesc . Nao há pressas

Reply:
@isabelcosta, tentei bastante fazer o que pedia, mas não consegui mesmo. Todas as soluções que vi preciso passar o mock das traduções, ele não vai diretamente buscar ao ficheiro e não reflete no snapshot :(

Reply:
@isabelcosta all clear :)